lib/rouge/lexers/perl6.rb

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+
+module Rouge
+  module Lexers
+    class Perl6 < Perl
+      title "Perl 6"
+      desc "The Perl 6 programming language (perl6.org)"
+
+      tag "perl6"
+      aliases 'pl6'
+
+      filenames '*.pl6', '*.pm6'
+      mimetypes 'text/x-perl6', 'application/x-perl6'
+
+      def self.analyze_text(text)
```

Comment:
The language detection has been changed in https://github.com/jneen/rouge/pull/763

No "scores" are used anymore.

Reply:
Thanks, so moving forward should I have a `detect?` method returning boolean instead?

Reply:
Yep!